validator/sawtooth_validator/journal/remme_batch_injector.py

```diff
@@ -97,7 +96,6 @@ def create_pay_reward_batch(self):
             SettingsView.setting_address(SETTINGS_MINIMUM_STAKE),
             SettingsView.setting_address(SETTINGS_BLOCKCHAIN_TAX),
             SettingsView.setting_address(SETTINGS_MIN_SHARE),
-            SettingsView.setting_address(SETTINGS_UNFREEZE_BONUS),
```

Comment:
Why? It is used for share formula and took here:
https://github.com/Remmeauth/remme-core/blob/811763fc596453f7f2b8f256f1b0f1265925b75d/remme/tp/consensus_account.py#L212

Reply:
Because we will use it's like constant not like setting value